CalibCalorimetry/EcalCorrelatedNoiseAnalysisAlgos/interface/TEcnaParPaths.h

```diff
@@ -75,12 +75,12 @@ class TEcnaParPaths : public TObject {
 
   Bool_t GetPaths();
 
-  TString ResultsRootFilePath();
-  TString ResultsAsciiFilePath();
-  TString HistoryRunListFilePath();
-  TString CMSSWBase();
-  TString CMSSWSubsystem();
-  TString SCRAMArch();
+  const TString &ResultsRootFilePath() const;
+  const TString &ResultsAsciiFilePath() const;
+  const TString &HistoryRunListFilePath() const;
+  const TString &CMSSWBase() const;
+  const TString &CMSSWSubsystem() const;
+  const TString &SCRAMArch() const;
```

Comment:
this is just a minor issue but I think this should either be 

> const TString & ResultsRootFilePath()

or

> const TString& ResultsRootFilePath()

Like it is now, "&" is attached to a function name which I see as a little bit confusing.